Scenes/Objects/closet.gd

```diff
@@ -0,0 +1,52 @@
+extends StaticBody2D
+class_name Closet
+
+var player: Player
+var is_player_hiding:bool = false
```

Comment:
inconsistent spacing 🤓

Reply:
Fixed. LGTM!

Reply:
you're not even the one pull requesting 😭